File: app/src/main/java/com/example/ordinacija/Data/PreglediResultVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class PreglediResultVM implements Serializable{
    public static class Row implements Serializable {
        public int PregledID;
        public Date Datum;
        public String Vrijeme;
        public String Opis;
        public int ZaposlenikID;
        public int PacijentID;
        public String Usluga;
        public Row(){

        }
    }
    public List<Row> rows;

}

File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/PreglediResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class PreglediResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int PregledID { get; set; }
            public DateTime Datum { get; set; }
            public string Vrijeme { get; set; }
            public string Opis { get; set; }
            public int ZaposlenikID { get; set; }
            public int PacijentID { get; set; }
            public string Usluga { get; set; }


        }
    }
}